src/collections/posts/installing-the-mod-guide-necessary-for-any-custom-songs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How To Download And Install New Custom Songs & Mods On Beat Saber
+section: speciality
+category: guide
+publish: 2024-03-22T00:00:00.000Z
+homepageText: null
+image: /static/uploads/posts/beat-saber-square.jpg
+showInPostListing: false
+---
+# If you haven’t played custom songs yet and are just getting started with Beat Saber, we’re excited for you!
+
+## You’re about to experience a whole WORLD of new content and the ability to experience “inside” your favorite music like only VR allows! It’s really a magical experience.
+
+The modding experience for computer-based VR (PCVR) and Meta/Oculus Quest standalone VR is a bit different. Be sure to read the right section for your headset!
+
+## PC VR:
+
+While mods are not *required* to play basic custom songs, mods can enhance your gameplay experience and unlock some nifty mapping effects. To get started, you’ll just need to download [**ModAssistant**](https://github.com/bsmg/ModAssistant/releases)
+
+<p align="center">
+    <img src="/uploads/mod-assistant-mods.png">
+</p>
+
+### Installing Mods With Mod Assistant
+
+1. First, [click here](https://github.com/bsmg/ModAssistant/releases) and click on the Latest release of ModAssistant.exe file to download
+
+<p align="center">
+    <img src="/uploads/mod-assistant-github.png">
+</p>
+
+This will download a small `ModAssistant.exe` file that you can place anywhere on your computer for future use and doesn’t require installing.
+
+2. When you first launch Mod Assistant, it will also try to automatically locate where you installed your game. If it can’t find it, you will have to manually locate the folder.
+
+    - Oculus User folder paths tend to be: `C:\Program Files\Oculus\Software\Software\hyperbolic-magnetism-beat-saber\`
+
+    - Steam User folder paths tend to be: `C:\Program Files (x86)\Steam\steamapps\common\Beat Saber\`
```

Comment:
Should "User" be capitalized in both bullet points? I could see why, but then I find it a bit odd that Folder is not also capitalized as they kinda form one "unit".